contrib/lib/src/json.rs

```diff
@@ -277,6 +278,12 @@ impl From<serde_json::Value> for JsonValue {
     }
 }
 
+impl<T> FromIterator<T> for JsonValue where serde_json::Value: FromIterator<T> {
+    fn from_iter<I: IntoIterator<Item=T>>(iter: I) -> Self {
+        JsonValue(iter.into_iter().collect())
```

Comment:
This `collect()` looks unnecessary and potentially expensive for large items -- `JsonValue(serde_json::Value::from_iter(iter))` should suffice.